### DISCUSSION TITLE
DeepMDKIT install

### DISCUSSION THREAD

**Discussion by 13yong** [deepmodeling/deepmd-kit#2149] [Q&A] [answered]
Hello developer, my CUDA version is 11.4 ,but when I run with 
# conda create -n deepmd deepmd-kit=2.1.5=*gpu libdeepmd=*=*gpu lammps cudatoolkit=11.3 horovod -c https://conda.deepmodeling.com -c defaults" ，
it is wrong.
But when I run with 
# conda create -n deepmd deepmd-kit=2.1.5=*gpu libdeepmd=*=*gpu lammps cudatoolkit=10.2 horovod -c https://conda.deepmodeling.com -c defaults",
It can work. But it is running at very low speed.

**Comment by njzjz** [comment, marked as answer]
We only provide 11.6.

**Reply by 13yong** [reply]
Thank you ! I got it.
